Brewfile

```diff
@@ -1,11 +1,4 @@
 # Homebrew dependencies to build Descent3
 
-# SDL
-brew "sdl2"
-
 brew "cmake"
-brew "googletest"
 brew "ninja"
-
-# zlib
-brew "zlib"
```

Comment:
We should preserve ability to install external dependencies without using vcpkg for users. Brewfile should left unchanged. You can override this in CI with direct brew invocation, i.e. `brew install cmake ninja`.

Reply:
While I'm all for a solution that enables using system-provided libraries, I'm not sure I agree that the Brewfile we provide should automatically install the build deps - if the user wants to install those deps via Brew then more power to them, but I think if our scripts install them then they should come from vcpkg.